src/components/views/settings/tabs/user/GeneralUserSettingsTab.js

```diff
@@ -22,6 +22,7 @@ import ProfileSettings from "../../ProfileSettings";
 import * as languageHandler from "../../../../../languageHandler";
 import SettingsStore from "../../../../../settings/SettingsStore";
 import LanguageDropdown from "../../../elements/LanguageDropdown";
+import SpellCheckSettings from "../../SpellCheckSettings";
```

Comment:
Feel free to sprinkle various 2021 Simon Brandner or foundation copyright lines across the various files you've modified. 😄